shared/services/rewards/utils.go

```diff
@@ -305,12 +304,39 @@ func DownloadRewardsFile(cfg *config.RocketPoolConfig, interval uint64, cid stri
 				}
 			}
 
+			deserializedRewardsFile, err := DeserializeRewardsFile(writeBytes)
+			if err != nil {
+				return fmt.Errorf("Error deserializing file %s: %w", rewardsTreePath, err)
+			}
+
+			// Get the original merkle root
+			downloadedRoot := deserializedRewardsFile.GetHeader().MerkleRoot
+
```

Comment:
```suggestion
			downloadedRoot := deserializedRewardsFile.GetHeader().MerkleRoot
			deserializedRewardsFile.GetHeader().MerkleRoot = ""

```

Alright hear me out on this one. Obviously, MerkleRoot gets overwritten on L316 when we call `generateMerkleTree()`, so why would we want to erase it first if it's about to get erased anyway?

Two reasons-
1. If the behavior changes, this is defensive
2. It improves the 'readability' of this function, since it makes it more obvious that the original header had a parsed root, we nuke it, and then it gets regenerated.

Reply:
Sounds good.